src/jit/gtlist.h

```diff
@@ -46,7 +46,7 @@ GTNODE(CNS_STR          , "sconst"       ,GenTreeStrCon      ,0,GTK_LEAF|GTK_CON
 //-----------------------------------------------------------------------------
 
 GTNODE(NOT              , "~"            ,GenTreeOp          ,0,GTK_UNOP)
-GTNODE(NOP              , "nop"          ,GenTree            ,0,GTK_UNOP)
```

Comment:
I don't think this is correct--IIRC we occasionally insert `NOP` nodes into a tree to confuse the optimizer (see e.g. `gtEffectiveVal`, which looks through `NOP`s).